tools/slintpad/package.json

```diff
@@ -18,9 +18,9 @@
         "lint:fix": "biome lint --fix",
         "clean": "rimraf dist dev-dist pkg",
         "start": "pnpm clean && pnpm build:wasm_lsp && pnpm start:vite",
-        "start:vite": "vite preview --open",
```

Comment:
This used to be the default port 3000, not 3001 as you set it now But that's just a nit.

Reply:
fixed